site/installing-scope.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Installing Weave Scope
+menu_order: 20
+---
+
+
+Weave Scope consists of three parts: the probe, the app and the user interface. It can be deployed either as a standalone configuration, or if you don’t want to manage the administration yourself, you can sign up for Weave Scope as a service.
+
+The following topics are discussed: 
+
+ * [Weave Scope Cloud Service](#signing-up-for-weave-scope-cloud-service)
```

Comment:
In general I think the structure should be:

```
* Installing Scope on Docker
  * Using Docker Compose
  * Using the Scope Service
* Installing Scope on Kubernetes
  * Using the Scope Service
* Installing Scope on Amazon ECS
  * Using the Scope Service
* Installing Scope on Mesosphere DC/OS
  * Using the Scope Service
```

I appreciate we want to funnel people towards the service at some point, but its not ready from prime time yet.  Lets make the top code block on the page the instructions on how to launch scope locally against docker, which is what we suspect the vast majority of users will need.